pkgs/by-name/po/poptracker/package.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, util-linux
+, SDL2
+, SDL2_ttf
+, SDL2_image
+, openssl
+, which
+, libsForQt5
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "poptracker";
+  version = "0.25.7";
+
+  src = fetchFromGitHub {
+    owner = "black-sliver";
+    repo = "PopTracker";
+    rev = "v${version}";
+    hash = "sha256-UqGEhs2AY9AKYrItmky3tcvXu/dXnnPwMXKwZ3klXcU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    SDL2
+    SDL2_ttf
+    SDL2_image
+    openssl
+    util-linux
+    makeWrapper
+  ];
```

```suggestion
  nativeBuildInputs = [
    makeWrapper
  ];
  
  buildInputs = [
    SDL2
    SDL2_ttf
    SDL2_image
    openssl
    util-linux
  ];
```
programs, hooks, which are used for the build go in nativeBuildInputs, things used by the build, like libraries to include / link to go in `buildInputs`